.github/workflows/pr-dev-preview.yml

```
@@ -0,0 +1,192 @@
+concurrency:
+  group: pr-dev-preview-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  PYTHON_VERSION: '3.11'
+  NODE_VERSION: '20'
+
+jobs:
+  start-build:
+    name: Trigger async build
+    if: >-
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)